sdks/python/apache_beam/examples/cookbook/datastore_wordcount.py

```diff
@@ -74,14 +74,14 @@
 from apache_beam.io import ReadFromText
 from apache_beam.io.datastore.v1.datastoreio import ReadFromDatastore
 from apache_beam.io.datastore.v1.datastoreio import WriteToDatastore
+from apache_beam.metrics import Metrics
 from apache_beam.utils.pipeline_options import GoogleCloudOptions
 from apache_beam.utils.pipeline_options import PipelineOptions
 from apache_beam.utils.pipeline_options import SetupOptions
 
-empty_line_aggregator = beam.Aggregator('emptyLines')
-average_word_size_aggregator = beam.Aggregator('averageWordLength',
-                                               beam.combiners.MeanCombineFn(),
-                                               float)
+empty_line_counter = Metrics.counter('main', 'emptyLines')
```

Comment:
While changing this, could you change the metric names to a standard format. Some files use 'empty_lines' format, this file use 'emptyLines'. I think we should standardize on the first.